tests/mmm/test_transformers.py

```diff
@@ -343,6 +344,32 @@ def test_logistic_saturation_min_max_value(self, x, lam):
         assert y_eval.max() <= 1
         assert y_eval.min() >= 0
 
+    def test_inverse_scaled_logistic_saturation_lam_half(self):
+        x = np.array([0.5] * 100)
+        y = inverse_scaled_logistic_saturation(x=x, lam=0.5, eps=np.ln(3))
+        expected = np.array([0.5] * 100)
+        np.testing.assert_almost_equal(
+            y.eval(),
+            expected,
+            decimal=5,
+            err_msg="The function does not behave as expected at lambda 0.5.",
+        )
+
+    @pytest.mark.parametrize(
+        "x, lam",
+        [
+            (np.ones(shape=(100)), 0.5),
+            (np.linspace(start=0.0, stop=1.0, num=50), 10),
+            (np.linspace(start=200, stop=1000, num=50), 0.001),
+            (np.zeros(shape=(100)), 1),
+        ],
+    )
+    def test_inverse_scaled_logistic_saturation_min_max_value(self, x, lam):
+        y = inverse_scaled_logistic_saturation(x=x, lam=lam)
```

Comment:
```suggestion
x = np.array([0, 1, 100, 500, 5000])
lam = np.array([...])[:, None]
y = inversed_scaled_logistic_saturation(x=x, lam=lam)
```

Lets remove the parametrize and make used of vectorized inputs. Think it is a bit more clear and consise

Reply:
done!

Reply:
Lam needs actual values though. 
The ... was place holder. 
Maybe np.array([0.01, 0.25, 0.75, 1.5, 5.0, 10.0, 15.0])

Reply:
sorry, read too fast and assumed that it would generate an array full of "None".
in that case, wouldn't it be better to stick to parametrize, for consistency with `test_logistic_saturation_min_max_value`?
in other words, why should they be different?

Reply:
I was interpreting this test as checking result domain being between 0 and 1 for various x and lam. We can just make use of the vectorized inputs which, imo, is a bit more clear and concise than the parameterized.

Reply:
ok, I have just updated it for the vectorized version